### PR TITLE
1267 remove feedback feature

### PR DESCRIPTION
**Pull request by Vainakh**

### CODE REVIEW

File: app/pages/OrganizationListingPage.tsx
Comment (richardxia) [inline app/pages/OrganizationListingPage.tsx:58]: On the lines just above this, could you remove the `"feedback"` string from both arrays? I can't comment directly on the lines, since they're just beyond the range of what the GitHub interface allows me to comment on, but they're just above this line.

File: app/pages/ServiceListingPage.tsx
Comment (richardxia) [inline app/pages/ServiceListingPage.tsx:64]: Similar comment about removing `"feedback"` on the lines above this.

File: app/components/listing/FeedbackModal.tsx
Comment (richardxia) [inline app/components/listing/FeedbackModal.tsx:20]: These are CSS classes that are actually still defined in their corresponding CSS file. Could you remove the following lines here? https://github.com/ShelterTechSF/askdarcel-web/blob/f6db93aedb298a8a0d2d3069fed1a94b6bf7f5f4/app/components/listing/ActionBar.scss#L39-L71

**Review by richardxia — COMMENTED** [review]
Thanks for submitting this! There are a few more places where you need to remove references to the feedback feature, but this otherwise looks great to me!

**Review by richardxia — APPROVED** [review]
Thanks for making this change! This PR LGTM now. Our usual process is that once a PR has been approved, the submitter is the one who hits the merge button, so feel free to do that at your leisure.

**Comment by Vainakh** [comment]
Richard,

Hitting that merge button felt good! Looking forward to many more hits or
buttons :) whichever it may be!

Thank you!

**Comment by Vainakh** [comment]
Hi Richard,

Should I start working on any other ticket?

Thanks!

**Comment by schroerbrian** [comment]
Thanks for your contribution, Adlan! I think this ticket would be a good one for you to tackle: https://github.com/ShelterTechSF/askdarcel-web/issues/1313

**Comment by Vainakh** [comment]
Brian,

Looking good!

Thank you!

**Pull request merged**
